code/solr/Solr.php

```diff
@@ -136,7 +137,7 @@ public static function service($core = null)
 
     public static function get_indexes()
     {
-        return FullTextSearch::get_indexes('SolrIndex');
+        return FullTextSearch::get_indexes('SilverStripe\FullTextSearch\Solr\SolrIndex');
```

Comment:
Completely agree with this since we are supporting php5.6 as the lowest version.
Will update code to use php5.5 class name resolution operator.